flytekit/bin/entrypoint.py

```diff
@@ -226,7 +226,25 @@ def setup_execution(
         tmp_dir=user_workspace_dir,
     )
 
-    if cloud_provider == _constants.CloudProvider.AWS:
+    # This rather ugly condition will be going away with #559. We first check the raw output prefix, and if missing,
+    # we fall back to the logic of checking the cloud provider. The reason we have to check for the existence of the
+    # raw_output_data_prefix arg first is because it may be set to None by execute_task_cmd. That is there to support
+    # the corner case of a really old propeller that is still not filling in the raw output prefix template.
+    if raw_output_data_prefix:
```

Comment:
Can we put a comment that points
To the other PR and says this will be deleted

Reply:
that is already in the comment right above this line.